# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/brotli/hotfix-pkgconfig.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- b/scripts/libbrotlicommon.pc.in
+++ a/scripts/libbrotlicommon.pc.in
@@ -7,5 +7,5 @@
 URL: https://github.com/google/brotli
 Description: Brotli common dictionary library
 Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -lbrotlicommon
-Libs: -L${libdir} -R${libdir} -lbrotlicommon
 Cflags: -I${includedir}
--- b/scripts/libbrotlidec.pc.in
+++ a/scripts/libbrotlidec.pc.in
@@ -7,6 +7,6 @@
 URL: https://github.com/google/brotli
 Description: Brotli decoder library
 Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -lbrotlidec
-Libs: -L${libdir} -R${libdir} -lbrotlidec
 Requires.private: libbrotlicommon >= 1.0.2
 Cflags: -I${includedir}
--- b/scripts/libbrotlienc.pc.in
+++ a/scripts/libbrotlienc.pc.in
@@ -7,6 +7,6 @@
 URL: https://github.com/google/brotli
 Description: Brotli encoder library
 Version: @PACKAGE_VERSION@
+Libs: -L${libdir} -lbrotlienc
-Libs: -L${libdir} -R${libdir} -lbrotlienc
 Requires.private: libbrotlicommon >= 1.0.2
 Cflags: -I${includedir}
